e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/okhttp3/WmHttpOkHttp3EMTest.kt

```diff
@@ -56,8 +56,8 @@ class WmHttpOkHttp3EMTest : SpringTestBase() {
 
                 assertHasAtLeastOne(solution, HttpVerb.GET, 200, "/api/wm/socketconnect/object", "OK")
             },
-            3
+            10
```

Comment:
hmmm... why this? unless very special cases, we shouldn't increase the timeout. why would it need 10 minutes? this might mask some performance issue. let's discuss it

Reply:
Forget to change.